manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml

```diff
@@ -6,33 +6,33 @@ metadata:
     opendatahub.io/categories: 'Getting started,Notebook environments'
 spec:
   displayName: Creating a Jupyter notebook
-  appName: jupyterhub
+  appName: jupyter
   durationMinutes: 5
-  icon: 'images/jupyterhub.svg'
+  icon: 'images/jupyter.svg'
   description: This quick start will walk you through creating a Jupyter notebook.
   introduction: |-
     ### This quick start shows you how to create a Jupyter notebook.
     Open Data Hub lets you run Jupyter notebooks in a scalable OpenShift hybrid cloud environment.
 
-    We have configured the JupyterHub spawner to support your amazing Data Scientist explorations and model deployments.
+    We have configured the Jupyter spawner to support your amazing Data Scientist explorations and model deployments.
 
     We just know you will absolutely love this amazing environment.  This quick start will get you working in a notebook in just a few minutes.
   tasks:
-    - title: Launch JupyterHub
+    - title: Launch Jupyter
       description: |-
-        ### To find the JupyterHub Launch action:
+        ### To find the Jupyter Launch action:
         1. Click **Applications** &#x2192; **Enabled**.
-        2. Find the JupyterHub card.
-        3. Click **Launch** on the JupyterHub card to access the JupyterHub **Start a notebook server** page.
+        2. Find the Jupyter card.
```

Comment:
Jupyter card / tile (applications - enabled) won't be there anymore in future, goes in the same direction regarding content that @DaoDaoNoCode has already hinted towards.

Reply:
> Jupyter card / tile (applications - enabled) won't be there anymore in future, goes in the same direction regarding content that @DaoDaoNoCode has already hinted towards.

@shalberd You've mentioned this several times, but would that be the case? I know we've had conversations regarding the rework of the jupyter section and so, but I cannot recall the part on where we decided to deprecate the jupyter tile

Reply:
@lucferbux there was always talk of "disable the notebook spawner through the Jupyter tile", `notebookController.enabled=false/true` but maybe the goal in the end is not the phase out the jupyter tile.

So you are correct, I might have misunderstood that, had a few conversations regarding jupyter tile vs Data Science Projects with @andrewballantyne on Slack. Cannot find the exact wordings anymore, in any case, if you want to keep the Jupyter tile, it is fine by me, just noticing an extreme focus on Data Science Projects. What's more, if you labeled the central opendatahub namespace with `'opendatahub.io/dashboard': 'true'`, it would probably show up in Data Science Projects, too, so I do not see the duality between central dashboard namespace / jupyter tile and data science projects being necessary. @andrewballantyne can say more after his vacation.

Reply:
Ok, yes, to be fair, we've had several conversations about the current state of the jupyter tile.
As of today, I think we are keeping the tile, so we might wanna review it and the related docs (quickstart and tutorials) just to reflect the change and remove all info about jupyterhub, but let's just limit the scope there and then we can have conversations about the future of the tile.

Reply:
Ok, I also changed that content for Melissa at https://github.com/opendatahub-io/odh-dashboard/issues/1616. Meaning Jupyter card reference is o :-)